apiary-metastore-events/kafka-metastore-events/kafka-metastore-integration-tests/src/test/java/com/expediagroup/apiary/extensions/events/metastore/kafka/messaging/KafkaApiaryMetastoreIntegrationTest.java

```diff
@@ -69,9 +71,7 @@
 @RunWith(MockitoJUnitRunner.class)
 public class KafkaApiaryMetastoreIntegrationTest {
 
-  @ClassRule
-  public static final SharedKafkaTestResource KAFKA = new SharedKafkaTestResource();
-
+  private static final KafkaContainer kafkaContainer = new KafkaContainer();
```

Comment:
For whatever reason in my testing I could not get this to run properly & consistently. I instead pivoted this to use kafka in a docker container via test containers instead.

Reply:
OK, what kind of errors were you getting? How does this approach differ? Slower?

Reply:
The error I was getting IIRC was kafka broker/read timeouts internal to the SharedKafkaTestResource on startup. 

The approach w/ testcontainers encapsulates the kafka executable in docker so it isolates it a bit more from the local system / jvm. There is a bit more overhead w/ the startup, but I can't really speak to if its slower/faster to the previous lib since it wasn't working at all for me

Reply:
OK, fine, well, let's stick with this if it works better, I'd rather have it working and a bit slower than not working but fast!